pkg/kubectl/rollout_status.go

```diff
@@ -126,7 +126,7 @@ func (s *StatefulSetStatusViewer) Status(namespace, name string, revision int64)
 		return "", false, err
 	}
 	if sts.Spec.UpdateStrategy.Type == apps.OnDeleteStatefulSetStrategyType {
-		return "", true, fmt.Errorf("%s updateStrategy does not have a Status", apps.OnDeleteStatefulSetStrategyType)
+		return "", true, fmt.Errorf("Status is not available for the OnDelete strategy type")
```

Comment:
I think we should use `!=` instead, i.e.
```go
	if sts.Spec.UpdateStrategy.Type != apps.RollingUpdateStatefulSetStrategyType {
      return "", true, fmt.Errorf("rollout status is only available for %s strategy type", apps.RollingUpdateStatefulSetStrategyType)
```

Note that errors should not be capitalized, because it's usually appended to another error like this `fmt.Errorf("failed to do this: %v", err)`. Using `rollout status` instead of `status` to avoid users confusing `status` command with `status` API field. 

Please also update the error message in DaemonSet to make them consistent.

Reply:
@janetkuo PR is updated.